lib/index.js

```diff
@@ -37,8 +38,7 @@ var _settings = exports._settings = {
 
   // This is a node-cron cron job. This will re-fetch the configuration on
   // a set interval, based on the cron-job configuration
-  cronJob: null,
-
+  cronJob: null
```

Comment:
It's only an error according to the ECMA 5 standard, so it's bad in .json files, but not in .js. I tend to leave them in all of my non-json-file objects, unless I have to support IE8. It makes subsequent additions to those objects have a cleaner diff. I would much rather leave these in. There's a setting you can set to have it not error out in WebStorm, I hope.